README.md

```diff
@@ -32,6 +33,31 @@ Read the [Changelog](https://github.com/Microsoft/vscode-go/blob/master/CHANGELO
 - [Code of Conduct](#code-of-conduct)
 - [License](#license)
 
+## We are moving!
+
+🏡 🚚 🏡 
+
+Our new home will be https://github.com/golang/vscode-go.
+
+As the extension grows in popularity and supports more features, it requires more maintenance and support.
+Since the introduction of Go modules, we have been collaborating with the Go team for module support.
+In 2019, we added support for `gopls`, the [Go language server](https://golang.org/s/gopls/README.md) which has
+resulted in closer collaboration with the Go team. More exciting work is on the horizon with the
```

Comment:
One more rephrasing (the word "support" was in there too much):

As the extension grows in features and popularity, it requires more maintenance and support. Since the introduction of Go modules, we have been collaborating with the Go team, particularly on support for `gopls`, the [Go language server](https://golang.org/s/gopls/README.md).

Reply:
Done with https://github.com/microsoft/vscode-go/pull/3237/commits/a697583f57edd82c4db828735079a3ef8f664a6f